kratos/tests/test_radial_basis_functions_utility.py

```diff
@@ -0,0 +1,105 @@
+            }
+        }"""))
+    gid_output.ExecuteInitialize()
+    gid_output.ExecuteBeforeSolutionLoop()
+    gid_output.ExecuteInitializeSolutionStep()
+    gid_output.PrintOutput()
+    gid_output.ExecuteFinalizeSolutionStep()
+    gid_output.ExecuteFinalize()
+
+class TestRadialBasisFunctionsUtility(KratosUnittest.TestCase):
+
+    def setUp(self):
+        # Create the test model part
+        self.model = KratosMultiphysics.Model()
+        self.main_model_part = self.model.CreateModelPart("MainModelPart")
+        self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.TEMPERATURE)
+
+        # Generate the problem domain
+        problem_domain = KratosMultiphysics.Quadrilateral2D4(
+            KratosMultiphysics.Node(1, 0.0, 0.0, 0.0),
+            KratosMultiphysics.Node(2, 0.0, 1.0, 0.0),
+            KratosMultiphysics.Node(3, 1.0, 1.0, 0.0),
+            KratosMultiphysics.Node(4, 1.0, 0.0, 0.0))
+
+        mesh_parameters = KratosMultiphysics.Parameters("{}")
+        mesh_parameters.AddEmptyValue("element_name").SetString("Element2D3N")
+        mesh_parameters.AddEmptyValue("create_skin_sub_model_part").SetBool(False)
+        mesh_parameters.AddEmptyValue("number_of_divisions").SetInt(5)
+
+        KratosMultiphysics.StructuredMeshGeneratorProcess(problem_domain, self.main_model_part, mesh_parameters).Execute()
+
+    def test_radial_basis_functions_utility_1x1_square(self):
+        # Create the sample points array
+        n_nodes = self.main_model_part.NumberOfNodes()
+        pts_coord = KratosMultiphysics.Matrix(n_nodes, 3, 0.0)
+        i = 0
+        for node in self.main_model_part.Nodes:
+            pts_coord[i,0] = node.X
+            pts_coord[i,1] = node.Y
+            i += 1
+
+        # Calculate the MLS shape functions in the square midpoint
+        midpoint = KratosMultiphysics.Vector(3)
+        midpoint[0] = 0.5
+        midpoint[1] = 0.5
+        midpoint[2] = 0.0
+        N_container = KratosMultiphysics.Vector()
+        h = 1 #Shape parameter for the kernel function (radial basis function, inverse multiquadratic)
+        InputShapeParameter = False
+        if InputShapeParameter:
+            KratosMultiphysics.RadialBasisFunctionsUtility.CalculateShapeFunctions(
+                pts_coord,
+                midpoint,
+                h,
+                N_container)
+        else:
+            KratosMultiphysics.RadialBasisFunctionsUtility.CalculateShapeFunctions(
+                pts_coord,
+                midpoint,
+                N_container)
+
+        # Save the obtained results in TEMPERATURE variable and visualize
+        output_results = False
+        if output_results:
+            i = 0
+            for node in self.main_model_part.Nodes:
+                node.SetSolutionStepValue(KratosMultiphysics.TEMPERATURE, 0, N_container[i])
+                i += 1
+            PostProcess(self.main_model_part, "test_radial_basis_functions_utility_1x1_square")
```

Comment:
Is this necessary for the test or just for debugging?

If the latter, I would comment it out or remove it altogether.

Reply:
It is not necessary, we can comment it.